doc/man3/SSL_CTX_set_client_hello_cb.pod

```diff
@@ -40,14 +45,9 @@ function, the ClientHello callback will be called again, and, if it returns
 success, normal handshake processing will continue from that point.
 
 SSL_client_hello_isv2() indicates whether the ClientHello was carried in a
```

Comment:
The declaration of this function is removed by your changesset, no ?

Reply:
No, as the rest of the changes you're commenting on says, it now always returns 0.

Reply:
It's also been deprecated